packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -72,9 +72,11 @@ export default defineComponent<KottiNavbar.PropsInternal>({
 		NavbarNotification,
 		NavbarQuickLink,
 	},
+	// @ts-expect-error: official declarations of @types/vue-clickaway do
+	// not satisfy the typechecker apparently
 	mixins: [clickaway],
 	props: makeProps(KottiNavbar.propsSchema),
-	setup(props, { emit }) {
+	setup(props: KottiNavbar.PropsInternal, { emit }) {
```

Comment:
In case somebody is wondering, vetur for some reason decided that props is `unknown` here,  I guess because. of the nonempty `mixins`. somehow, since this happens only for components that rely on 'vue-clickaway'